File: LENGUAJES/PHP/Ejercicio1.php
<?php
// BUCLES FOR

function mostrarPalabra() {
    // Solicitar al usuario que ingrese una palabra
    echo "Ingrese la palabra que quiere que se repita: ";
    $palabra = trim(fgets(STDIN));
    
    // Validar que la palabra no esté vacía
    while (empty($palabra)) {
        echo "La entrada no puede estar vacía. Inténtalo de nuevo.\n";
        echo "Ingrese la palabra que quiere que se repita: ";
        $palabra = trim(fgets(STDIN));
    }
    
    // Solicitar al usuario cuántas veces quiere que se repita la palabra
    do {
        echo "¿Cuántas veces quiere que se repita la palabra? ";
        $input = trim(fgets(STDIN));
        $repeticiones = intval($input);
        
        if ($repeticiones <= 0) {
            echo "Por favor, ingrese un número positivo.\n";
        }
    } while ($repeticiones <= 0);
    
    // Imprimir la palabra la cantidad de veces especificada
    for ($i = 0; $i < $repeticiones; $i++) {
        echo $palabra . "\n";
    }
}
mostrarPalabra();
?>

File: LENGUAJES/GO/Ejercicio1.go
//BUCLES FOR
package main

import (
	"fmt"
	"strconv"
	"strings"
)

func main() {
	mostrarPalabra()
}

func mostrarPalabra() {
	var palabra string
	fmt.Print("Ingrese la palabra que quiere que se repita: ")
	fmt.Scanln(&palabra)
	palabra = strings.TrimSpace(palabra)


	for palabra == "" {
		fmt.Println("La entrada no puede estar vacía. Inténtalo de nuevo.")
		fmt.Print("Ingrese la palabra que quiere que se repita: ")
		fmt.Scanln(&palabra)
		palabra = strings.TrimSpace(palabra)
	}

	var repeticiones int
	for {
		fmt.Print("¿Cuántas veces quiere que se repita la palabra? ")
		var input string
		fmt.Scanln(&input)
		var err error
		repeticiones, err = strconv.Atoi(input)
		if err != nil || repeticiones <= 0 {
			fmt.Println("Entrada no válida. Por favor, ingrese un número entero positivo.")
		} else {
			break
		}
	}

	for i := 0; i < repeticiones; i++ {
		fmt.Println(palabra)
	}
}